src/test/test_confparse.c

```diff
@@ -906,11 +906,14 @@ test_confparse_unitparse(void *args)
   tt_assert(ok);
 
   /* u64 overflow */
-  /* XXXX our implementation does not currently detect this. See bug 30920. */
-  /*
   tt_u64_op(config_parse_memunit("20000000 TB", &ok), OP_EQ, 0);
   tt_assert(!ok);
-  */
+  // Fails double comparison
+  tt_u64_op(config_parse_memunit("15000000.5 TB", &ok), OP_EQ, 0);
```

Comment:
This test fails the double check as the float representing 15000000.5 TB is greater than `(double) INT64_MAX`

Reply:
Can you put this information in the code?